magento/magento2
# ISSUE-27397 - Fix phrase handling in widget validation

### Description (*)
The doc block in \Magento\Widget\Model\Widget\Instance::validate indicates that it can return a boolean or phrase object, but \Magento\Widget\Controller\Adminhtml\Widget\Instance\Validate doesn't handle the phrase object properly. This PR updates the controller to recognize a phrase result value as an error message and output it to the user.

### Fixed Issues (if relevant)
1. Fixes magento/magento2#27397

### Manual testing scenarios (*)
1. Create a custom widget type (in code).
2. Ensure that the widget class implements the validate() method and returns an instance of \Magento\Framework\Phrase.
3. In admin, go to Content > Widgets.
4. Click Add Widget.
5. Select the custom widget type and save.
6. Instead of saving, an error message should appear with the text from the widget's validate method.

### Contribution checklist (*)
